src/utils/IDMassAccuracy.cpp

```diff
@@ -131,8 +131,11 @@ class TOPPIDMassAccuracy :
     registerIntOption_("number_of_bins", "<#bins>", 100, "Number of bins that should be used to calculate the histograms for the fitting.", false, true);
     setMinInt_("number_of_bins", 10);
 
-    registerStringOption_("generate_gnuplot_scripts", "<false>", "false", "If this option is set to true, the distributions and the fits are used to generate a gnuplot script, that can be used to generate plots. The options 'precursor_out' and 'fragment_out' must be set to take this effect.", false, true);
-    setValidStrings_("generate_gnuplot_scripts", ListUtils::create<String>("true,false"));
+    registerOutputFile_("precursor_fit_out", "<file>", "", "Gaussian fit of the frequency of the deviations from the precursors. The options 'precursor_out' and 'fragment_out' must be set to take this effect.", false, true);
```

Comment:
canonically, this should be called `out_precursor_fit`.
The description should say `"Gaussian fit to the histogram of mass deviations from the precursors. The option 'precursor_out' must be set for this file to be generated."

The current behaviour is a bit unfortunate, because no file gets written if `precursor_out` (should be `out_precursor` but hey...) is not written. Usually TOPP tools do not have this weird dependency.
Solution: compute internal data if any of those two flags is requested.